packages/wrangler/src/pages/publish.tsx

```diff
@@ -389,3 +415,21 @@ export const Handler = async ({
 	);
 	await metrics.sendMetricsEvent("create pages deployment");
 };
+
+function validateRoutesFile(_routes: string, routesPath: string) {
+	const routes = JSON.parse(_routes);
+
+	if (!isRoutesJSONSpec(routes)) {
+		throw new FatalError(
+			`Invalid _routes.json file found at: ${routesPath}. Please make sure the JSON object has the following format:
+      {
+        version: ${ROUTES_SPEC_VERSION};
+        include: string[];
+        exclude: string[];
+      }
+			and that all rules contain at least one entry.
```

Comment:
"and that all rules contain at least one entry." isn't quite right - let's change it to "and that at least one include rule is provided."